lldb/test/Shell/SymbolFile/DWARF/x86/improperly_nested_blocks.s

```diff
@@ -0,0 +1,100 @@
+## This test checks that lldb handles (corrupt?) debug info which has improperly
+## nested blocks. The behavior here is not prescriptive. We only want to check
```

Comment:
Could you add comments somewhere in this file that note where the "improper" bits are?

Somewhere in the block where you use the `.L` is probably best.